app/routes.py

```diff
@@ -1,14 +1,18 @@
-from flask import render_template
-from app import app
-
-@app.route('/')
-def index():
-	return render_template('login_page.html')
-
-@app.route('/homepage')
-def homepage():
-	return render_template('logout_page.html')
-
-
-
-
+from flask import render_template, flash, redirect, url_for
+from app import app
+from app.forms import LoginForm
+
+@app.route('/')
+@app.route('/test')
+def test():
+    user = {'username' : 'User'}
+    return render_template('test.html', title = 'Home', user = user)
+
+@app.route('/login', methods=['GET','POST'])

```

Comment:
Make this route ('/') that way it is the first page seen when viewing this

Reply:
Just made changes which can be viewed now.